features/steps/step_checking_status.py

```diff
@@ -0,0 +1,61 @@
+from behave import when, then
+from selenium import webdriver
+from selenium.webdriver.support.wait import WebDriverWait
+from helpers.icon_in_div_changed import IconInDivChanged
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.common.by import By
+
+
+FULL_NAME = None
```

Comment:
Does it make sense to store user1 full name in same place we store user1 login/pass?
"def before_all(context):
    context.hipchat_login = HIPCHAT_LOGIN"

Reply:
Thanks for comment. It's impossible because we should be logged in to get the full name from the settings page.

Reply:
Seems pretty possible to me to do this in some background step